### DISCUSSION TITLE
Mistake in Wiki-article about Aggregate Device?

### DISCUSSION THREAD

**Discussion by quadrup** [ExistentialAudio/BlackHole#670] [General]
In step "4." in the Wiki-article about "Aggregate Device" it says: 

> _**"IMPORTANT: Due to issues with macOS the Built-in Output must be enabled and listed as the top device in the Aggregate"**_ 

But in the screenshot picture right below that it shows that the **built-in Microphone is listed as the top device, not the built-in output**. I tried putting the built-in Microphone as first device and it actually works a lot better than with the built-in output as top device. 

Should the text of the above quote be changed or maybe the picture? It seems inconsistent.

**Comment by devinroth** [comment]
It should be either or. I updated it. Thanks.
